src/components/AddPlant.js

```diff
@@ -24,31 +24,37 @@ export const AddPlant = () => {
         'Content-Type': 'application/json',
         Authorization: accessToken
       },
-      // eslint-disable-next-line object-shorthand
-      body: JSON.stringify({ plantname: plantname, species: species })
+      body: JSON.stringify({ plantname, species })
```

Comment:
is this object descontructing?

Reply:
Not exactly, when I was changing things around in the code I got an ESLint error for this specific line that said "expected property shorthand". To my understanding, since the properties plantname and species match the variables you can use shorthand syntax. (This is just from googling)

Reply:
I trust you on this one